django/db/backends/oracle/validation.py

```diff
@@ -0,0 +1,38 @@
+from django.core import checks
+from django.db.backends.base.validation import BaseDatabaseValidation
+
+
+class DatabaseValidation(BaseDatabaseValidation):
+    def check_field(self, field, **kwargs):
+        """
+        Oracle doesn't support a database index on some data types.
+        """
+        errors = super().check_field(field, **kwargs)
+        # Ignore any related fields.
+        if getattr(field, 'remote_field', None):
```

Comment:
Perhaps the duplicated code between this and the MySQL backend suggests a need for some refactoring.